package.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "salesforcedx-vscode-capture-startup-performance",
+  "displayName": "Capture Extension Startup Performance",
+  "description": "Extension that captures the Extension Activation Stats from 'Developer: Startup Performance' and reports the data via telemetry",
+  "version": "1.0.0",
+  "publisher": "salesforce",
+  "repository": {
+    "url": "https://github.com/forcedotcom/salesforcedx-vscode-capture-startup-performance"
+  },
+  "icon": "images/VSCodeBundle.png",
+  "license": "BSD-3-Clause",
+  "engines": {
+    "vscode": "^1.86.0"
+  },
+  "categories": [
+    "Other"
+  ],
+  "main": "./out/src/index.js",
+  "dependencies": {},
+  "devDependencies": {
+    "@types/node": "^18.11.9",
+    "@types/jest": "^29.5.12",
+    "@types/vscode": "^1.86.0",
+    "@tsconfig/node18": "^18.2.4",
+    "eslint": "^8.57.0",
+    "@typescript-eslint/eslint-plugin": "^8.2.0",
+    "prettier": "^3.3.3",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-prettier": "^5.2.1",
+    "eslint-plugin-import": "^2.29.1",
+    "jest": "^29.7.0",
+    "esbuild": "^0.23.1",
+    "@types/shelljs": "^0.8.15",
+    "shelljs": "^0.8.5"
+  },
+  "packaging": {
+    "assets": [
+      "LICENSE",
+      "package.nls.json",
+      "README.md",
+      "images",
+      "dist"
+    ],
+    "packageUpdates": {
+      "main": "dist/index.js",
+      "dependencies": {
+        "applicationinsights": "^3.2.2",
```

Comment:
@daphne-sfdc why are these in package updates section?

Reply:
I got that from the `packaging` section of package.json of salesforcedx-vscode-core.
```
    "packageUpdates": {
      "main": "dist/src/index.js",
      "dependencies": {
        "applicationinsights": "1.0.7",
        "shelljs": "0.8.5"
      },
      "devDependencies": {}
    }
```